.eslintrc.json

```diff
@@ -178,7 +178,7 @@
         "object-curly-spacing": "error",
         "object-property-newline": "error",
         "object-shorthand": "error",
-        "one-var": "error",
+        "one-var": "off",
```

Comment:
What was the reason for disabling this rule?

Reply:
Because only allows one variable per method.